scripts/update-endpoints/generated/endpoints.json

```diff
@@ -495,6 +495,17 @@
     "previews": [],
     "renamed": null
   },
+  {
+    "method": "GET",
+    "url": "/repos/{owner}/{repo}/actions/permissions",
+    "documentationUrl": "https://docs.github.com/rest/reference/actions#get-github-actions-permissions-for-a-repository",
+    "parameters": [
+      { "alias": null, "deprecated": null, "in": "PATH", "name": "owner" },
+      { "alias": null, "deprecated": null, "in": "PATH", "name": "repo" }
+    ],
+    "previews": [],
+    "renamed": { "note": null }
```

Comment:
The `renamed` was lost, here and in a few other cases below

Reply:
ah that's actually by design, the note was useless, it can be derived from the other properties